### PR TITLE
Feature/deploy gallery

### PR DESCRIPTION
**Pull request by Keith-Go**
## Ticket
- Setup Deploy Github Action

### CODE REVIEW

File: gallery_app/.env
Comment (krish-do-goldenowl) [inline gallery_app/.env:1]: remove if no use

File: gallery_app/android/fastlane/Appfile
Comment (krish-do-goldenowl) [inline gallery_app/android/fastlane/Appfile:1]: remove if no use

File: gallery_app/android/fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+default_platform(:android)
+
+platform :android do
+  desc "Runs all the tests"
+  lane :test do
+    gradle(task: "test")
+  end
+
+  desc "Submit a new Beta Build to Crashlytics Beta"
```
Comment (krish-do-goldenowl) [inline gallery_app/android/fastlane/Fastfile]: `Crashlytics` -> `Firebase Distributed`

File: gallery_app/android/fastlane/README.md
Comment (krish-do-goldenowl) [inline gallery_app/android/fastlane/README.md:1]: update this file based on this project => it seems you just copied/past the file

**Comment by krish-do-goldenowl** [comment]
@Keith-Go this is the wrong way of implementing GitHub Action => We use Fastlane or GithubAction
Please set up the Github Action property later (when you begin demo 01)

**Comment by Keith-Go** [comment]
Resolved

**Comment by luke-nguyen-goldenowl** [comment]
Approved

**Pull request merged**